test/functional/change_stream_spec_tests.js

```diff
@@ -116,6 +116,7 @@ describe('Change Stream Spec', function() {
 
       if (result.success) {
         value = EJSONToJSON(value);
```

Comment:
In my branch I made one extra change by replacing L37 above:
```
const specData = JSON.parse(specString);
```
with
```
const specData = EJSON.parse(specString, { relaxed: true });
```
which allows you to outright remove the `EJSONToJSON` method. I'm comfortable leaving that out of this change for now, but I think it's what we should ultimately do.